Esri/arcgis-runtime-samples-android
# New Sample: Add features with contingent values

## Description

PR to add a new Kotlion sample _"Add features with contingent values"_ to the repo

## Links and Data

Sample Epic: `runtime/common-samples/issues/3133`

## What To Review


-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files

## How to Test

Run the sample on the sample viewer or the repo. Test contingent value functionality by setting all the attribute values (or deliberately not) to see if the app handles it gracefully. 

## To Discuss

There could be scope to make this sample more _Kotliny_ since there are multiple uses of `MutableList` for contingency values and views being modified accordingly. So any helpful feedback here would be appreciated. 

## Screenshots
![demo](https://user-images.githubusercontent.com/14892474/161128036-8679cd42-7ef2-4feb-9110-0d97be71bf35.gif)
